### PR TITLE
fix: issues with icon positioning

### PR DESCRIPTION
**Pull request by damianmuti**
Changed positioning of the icon. This PR is currently for testing purposes on different environments, so I'll edit it once it's ready for CR.

**EDIT**: Finally fixed the issue. Please verify, @CarlosSampedro @andresmoschini @GusBaamonde 

image

https://cdn.fromdoppler.com/doppler-ui-library/build268/signup.html

### CODE REVIEW

**Comment by andresmoschini** [comment]
Thanks @damianmuti, could you add this element in some place in the playground to test it? (Or maybe it is already there and I cannot find it)

**Comment by damianmuti** [comment]
Sure, @andresmoschini . I was going to manually change the class name in the inspector for testing purposes, but I've just added a `secure-message` span on the signup page. 👍

File: assets/templates/signup.html [hunk 1]
```diff
@@ -115,7 +115,7 @@ <h5>email, automation & data marketing</h5>
                 <label for="password">Contraseña: <a toggle="#password-field" class="ms-icon icon-view show-hide"> <span class="content-eye">Mostrar</a></label>
                 <input type="password" name="password" id="password-field" placeholder="Escribe tu clave secreta">
                 <div class="wrapper-password">
-                  <p class="password-message"><span class="waiting-message">8 caracteres como mínimo</span><span class="waiting-message">Una letra</span><span class="waiting-message">Un número</span></p>
+                  <p class="password-message"><span class="secure-message">8 caracteres como mínimo</span><span class="waiting-message">Una letra</span><span class="waiting-message">Un número</span></p>
```
Comment (andresmoschini) [inline assets/templates/signup.html]: What do you think about adding both lines?

**Pull request merged**